go/private/actions/stdlib.bzl

```diff
@@ -53,19 +57,23 @@ def _should_use_sdk_stdlib(go):
             go.mode.link == LINKMODE_NORMAL)
 
 def _has_precompiled_stdlib(version_string):
+    minor = _minor_version(version_string)
+    return minor != None and minor < 20
+
+def _minor_version(version_string):
     if version_string[:2] != "1.":
-        return False
+        return None
     minor = version_string[2:]
     dot = minor.find(".")
     if dot != -1:
         minor = minor[:dot]
-    rc = minor.find("rc")
-    if rc != -1:
-        minor = minor[:rc]
-    beta = minor.find("beta")
-    if beta != -1:
-        minor = minor[:beta]
-    return int(minor) < 20
+    minor_digits = ""
+    for e in minor.elems():
+        if e.isdigit():
+            minor_digits += e
+        else:
+            break
+    return int(minor_digits)
```

Comment:
Can we reuse the new function in `common.bzl` here?

Reply:
Done